sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -162,6 +162,12 @@ def __init__(self, packages, options, environment_version):
             value=to_json_value(job_type)),
         dataflow.Environment.VersionValue.AdditionalProperty(
             key='major', value=to_json_value(environment_version))])
+    if job_type.startswith('FNAPI_'):
```

Comment:
<!--new_thread; commit:0df823c8839461643514dc954b59c2e073253fb0; resolved:0-->
Can we add a TODO before this section to factor out job types into some enum-like object?